### ISSUE TITLE
bad-format-string false positive

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#296]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

From stackoverflow:

```
#!python

"""hello I am the module spam.py"""
from __future__ import unicode_literals
'hello {world}'.format(world='potato')
```

This emits a bad-format-string on Python 2.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/296
